package.json

```diff
@@ -12,7 +12,7 @@
     "cmd:build": "microbundle --external all --sourcemap false -f es,cjs -i ",
     "build:core": "yarn cmd:build src/index.js",
     "build:helper": "yarn cmd:build src/helpers.js -o helpers/index.js",
-    "build": "npm-run-all --parallel build:*",
+    "build": "npm-run-all --parallel build:* && cp typing/index.d.ts dist/index.d.ts",
```

Comment:
This is better to add a "build:copy-types" I think:
1. This is less dependent of the OS/Shell
2. It works with npm-run-all (parallel execution here)

Reply:
Ok, will do ;)

Reply:
@fabienjuif do you want to support windows development on the library? If so, I will add a nodejs script to copy the file, if not I'll just leave it that way :)

Reply:
What do you think?

We can wait that someone struggle with it if you don't want to look into this.
If you have time, maybe fix it right away is a better solution.
As you wish @AFaugeras 👌